File: 2-May Jewels and Stones/Jewels_and_Stones_1.c
int numJewelsInStones(char * J, char * S){

    int i, j, jwl = 0;
    for(i = 0; i < strlen(J); i++){
        //printf("%c",J[i]);
        for(j = 0; j < strlen(S); j++){
            if(S[j] == J[i])
                jwl++;
        }
    }
    return jwl;
}

File: 02-May Jewels and Stones/Jewels_and_stones_1.py
class Solution:
    def numJewelsInStones(self, J: str, S: str) -> int:
        if S is None:
            return 0
        if J is None:
            return 0
        ct=0
        for i in S:
            if i in J:
                ct+=1
        return ct
